shared/AppInsightsCore/src/JavaScriptSDK/EnvUtils.ts

```diff
@@ -29,6 +31,7 @@ const strReactNative = "ReactNative";
 const strMsie = "msie";
 const strTrident = "trident/";
 const strXMLHttpRequest = "XMLHttpRequest";
+const STR_REDACTED = "REDACTED";
```

Comment:
Constants are usually added in a constants file in the project instead, in this case InternalConstants file